jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/verification/VerifierResolverImpl.java

```diff
@@ -26,4 +27,12 @@ public Verifier getVerifierByName(String name) {
             .orElseThrow(() -> new NoSuchBeanDefinitionException(name));
     }
 
+    public Verifier getVerifierByName(String name, String defaultVerifierName) {
```

Comment:
default verifier should be of appropriate type - not string.

Reply:
Its verifier name thats why String.
And it is not Verifier because all of our verifiers (equals, contains etc) are in internal package so the users could not use them in their steps. From my POV its better to move VerifierNames to public API, than to expose all of our internal verifiers.

Reply:
Spring doesn't care what name has your package as long as it is present in component scan. Default verifier must not be resolved, but provided.

Reply:
But you would be importing from internal, and there is no guarantee that the classes in internal wont be moved to different package, or renamed/replaced. 

E.g. we guarantee to users that some verifier with name EQ is present, but not that class org.jbehavesupport.core.internal.verification.EqualsVerifier will exist.

Reply:
Discussed - in the end decided we will use appropriate type.